Microsoft.WindowsAzure.Storage/src/executor.cpp

```diff
@@ -297,6 +298,10 @@ namespace azure { namespace storage { namespace core {
                         retryable_exception = e.retryable();
                         throw;
```

Comment:
If the `ex_ptr` is not set here, I believe all `storage_exception` will not be available when retrieving unhandled exception.

Reply:
discussed this offline yesterday, for now storing nonstorage exceptions